atomic_reactor/schemas/config.json

```diff
@@ -607,6 +607,17 @@
         "items": {
             "type": "string"
         }
+    },
+    "source_containers": {
+        "description": "Configuration options for building source container images",
+        "type": "object",
+        "properties": {
+            "source_image_builder": {
```

Comment:
Wonder if 'image' should be the last word in this name, something like source_builder_image.